ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-stable.yaml

```diff
@@ -59,6 +59,11 @@ releases:
       architecture: amd64
       channel: candidate
       version: "4.13"
+  source:
```

Comment:
stable e2e ci does not run any test suite, so we do not need the setting here.

Reply:
it has openshift-upgrade-qe-test-arch-migration-upi which has pre step, so keep it.

Reply:
hmm, I guess that is some mis-configuration on the test chain, I can update it later in another PR.

Reply:
could you please make that PR after this PR is merged?
I am fear there are more updated introduced when waiting this PR merged.